src/internal/connector/exchange/service_iterators.go

```diff
@@ -18,14 +19,7 @@ import (
 	"github.com/alcionai/corso/src/pkg/services/m365/api"
 )
 
-type addedAndRemovedItemIDsGetter interface {
-	GetAddedAndRemovedItemIDs(
-		ctx context.Context,
-		user, containerID, oldDeltaToken string,
-		immutableIDs bool,
-		canMakeDeltaQueries bool,
-	) ([]string, []string, api.DeltaUpdate, error)
-}
+var ErrFolderNotFound = clues.New("folder not found")
```

Comment:
this doesn't seem to be used at all? If it is going to be used, should it be added to the set of errors that are exported to SDK consumers instead?

Reply:
Weird, I'm surprised the linter didn't catch that.